lib/galaxy/tool_util/deps/resolvers/__init__.py

```diff
@@ -5,7 +5,7 @@
     abstractmethod,
     abstractproperty,
 )
-
+import os.path
```

Comment:
We sort the imports in 3 groups: standard library, 3rd party and internal. Can you move this right after `import errno` please?